modules/chat/src/utils/index.ts

```diff
@@ -23,6 +25,64 @@ export async function validateUsersInput(grpcSdk: ConduitGrpcSdk, users: any[])
       });
     }
   }
+  return usersToBeAdded;
+}
+
+export async function sendInvitations(users: any, sender: User, room: ChatRoom, url: string, config: any, grpcSdk: ConduitGrpcSdk) {
```

Comment:
Add type for users.
Pass a `sendEmail` boolean arg instead of the whole module config.